src/helpers/actions.ts

```diff
@@ -37,7 +39,16 @@ export async function getProposal(space, id) {
   return proposal;
 }
 
-export async function getSpace(id: string, includeDeleted = false) {
+export async function getSpace(id: string, includeDeleted = false, network = defaultNetwork) {
+  if (network !== defaultNetwork) {
+    const spaceExist = await sxSpaceExists(id);
+    if (!spaceExist) return false;
+
+    return {
+      network: 1
```

Comment:
feel like this may trouble us in the future. 😢

Reply:
We could pass `0` instead

Reply:
hmm if it works then yea 😄